pkg/render/logstorage.go

```diff
@@ -425,7 +376,6 @@ func (es *elasticsearchComponent) Objects() ([]client.Object, []client.Object) {
 			toCreate = append(toCreate, secret.ToRuntimeObjects(secret.CopyToNamespace(ElasticsearchNamespace, es.cfg.KeyStoreSecret)...)...)
 		}
 		toDelete = append(toDelete, es.kibanaCR())
-		toDelete = append(toDelete, es.curatorCronJob())
```

Comment:
We might still want to include logic to delete curator from the cluster, since otherwise I think upgraded clusters will continue to have a downlevel curator component installed.

Reply:
- service account
- any rbac
- the job itself
- any other supporting resources